### PR TITLE
Docs: Reverted documentation changes according to meet standard

### LINKED ISSUE #204: DOC - reverted examples as Timo
#203

### PR DESCRIPTION
**Pull request by Dudleyneedham**
## fixes #204 

Created a new branch with the latest develop. 

reverting some documentation on the latest develop.

## How to test:

Documentation

## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [x] I have [left the code in a better state](https://deviq.com/boy-scout-rule/)
- [x] I have documented the changes (where applicable)

### CODE REVIEW

File: src/requestforattestation/RequestForAttestation.ts [hunk 1]
```diff
@@ -62,15 +62,17 @@ function getHashRoot(leaves: Uint8Array[]): Uint8Array {
 
 export default class RequestForAttestation implements IRequestForAttestation {
   /**
-   * [STATIC] Builds an instance of [[RequestForAttestation]], from a simple object with the same properties.
```
Comment (tjwelde) [inline src/requestforattestation/RequestForAttestation.ts]: The static can stay

File: src/requestforattestation/RequestForAttestation.ts [hunk 2]
```diff
@@ -126,14 +127,19 @@ export default class RequestForAttestation implements IRequestForAttestation {
   }
 
   /**
-   * Removes [[Claim]] properties from the [[RequestForAttestation]] object, provides anonymity and security when building the [[createPresentation]] method.
```
Comment (tjwelde) [inline src/requestforattestation/RequestForAttestation.ts]: The added doc was good

**Comment by tjwelde** [comment]
Can you please also change the ````` @example ```javascript ````` part in https://github.com/KILTprotocol/sdk-js/blob/develop/src/attestedclaim/AttestedClaim.ts#L28 and https://github.com/KILTprotocol/sdk-js/blob/develop/src/attestation/Attestation.ts#L39 ?

File: src/requestforattestation/RequestForAttestation.ts (hunk 1 above)
Comment (Dudleyneedham) [reply src/requestforattestation/RequestForAttestation.ts]: I have added this

File: src/requestforattestation/RequestForAttestation.ts (hunk 2 above)
Comment (Dudleyneedham) [reply src/requestforattestation/RequestForAttestation.ts]: Updated accordingly

File: src/attestation/Attestation.ts [hunk 3]
```diff
@@ -108,11 +104,11 @@ export default class Attestation implements IAttestation {
 
   /**
    * [ASYNC] Stores the attestation on chain.
+   * Use [[store]] to store an attestation on chain, and to create an [[AttestedClaim]] upon success:
```
Comment (tjwelde) [inline src/attestation/Attestation.ts]: I think this should be a comment inside the example

File: src/attestedclaim/AttestedClaim.ts [hunk 4]
```diff
@@ -44,11 +43,10 @@ export default class AttestedClaim implements IAttestedClaim {
 
   /**
    * Builds a new [[AttestedClaim]] instance.
-   *
+   * Create an [[AttestedClaim]] upon successful [[Attestation]] creation:
```
Comment (tjwelde) [inline src/attestedclaim/AttestedClaim.ts]: Same as above

**Comment by Dudleyneedham** [comment]
Added the information as per request

**Pull request merged**